autoload/lsp/omni.vim

```diff
@@ -319,8 +328,9 @@ function! lsp#omni#get_vim_completion_items(options) abort
             \ 'empty': 1,
             \ 'icase': 1,
             \ }
-        if has_key(l:completion_item, 'textEdit') && type(l:completion_item['textEdit']) == s:t_dict && has_key(l:completion_item['textEdit'], 'newText')
+        if has_key(l:completion_item, 'textEdit') && type(l:completion_item['textEdit']) == s:t_dict && has_key(l:completion_item['textEdit'], 'range') && has_key(l:completion_item['textEdit'], 'newText')
```

Comment:
I've added the `range` check due to avoid the error from `InsertReplaceTextEdit`.

I will try to support the `InsertReplaceTextEdit` partially later.